molecule/default/converge.yml

```diff
@@ -0,0 +1,30 @@
+---
+- name: Converge
+  hosts: all
+  roles:
+    - role: ansible-django
+      django_system_user: "django_example_app"
+      django_system_user_home: "/home/{{ django_system_user }}"
+      django_recreate_virtual_env: false
+      django_codebase_path: "{{ django_system_user_home }}/app"
+      django_versioned_path: "{{ django_codebase_path }}-versioned"
+      django_checkout_path: "{{ django_versioned_path }}/{{ ansible_date_time['epoch'] }}"
+      django_git_url: "https://github.com/onaio/kaznet-web"
+      django_git_version: "master"
```

Comment:
Do you think it is a good idea to lock this to a tag or commit hash?

Reply:
Yes, that would be good. I'll change this in the latest commits